### DISCUSSION TITLE
Dynamic mesh creation is slow

### DISCUSSION THREAD

**Discussion by 1136386803** [bepu/bepuphysics1#31] [Q&A]

**Comment by RossNordby** [comment]
I'm assuming this is related to the earlier e-mail.

Given the age of the original codebase and the fact that you're using the fixed point fork (which I did not make nor do I maintain, for what it's worth) which takes a pretty big performance hit, you'll probably want to simplify the geometry or split it into more chunks that can be processed independently or in parallel.

**Comment by 1136386803** [comment]
Thank you, I have found the problem, I wrote the code has a problem
